<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test-20200124121031" tests="3" file=".py" time="17.152" timestamp="2020-01-24T12:10:50" failures="0" errors="0" skipped="0">
	<testcase classname="Test" name="test_login_form_validations" time="5.739" timestamp="2020-01-24T12:10:39" file="unittests\vaeative_unit.py" line="156">
		<system-out><![CDATA[logged in successfully
]]></system-out>
		<system-err><![CDATA[]]></system-err>
	</testcase>
	<testcase classname="Test" name="test_signup_form_validations" time="6.010" timestamp="2020-01-24T12:10:45" file="unittests\vaeative_unit.py" line="184">
		<system-out><![CDATA[signup in successfully
]]></system-out>
		<system-err><![CDATA[]]></system-err>
	</testcase>
	<testcase classname="Test" name="test_signup_tag_validations" time="5.403" timestamp="2020-01-24T12:10:50" file="unittests\vaeative_unit.py" line="174">
		<system-out><![CDATA[]]></system-out>
		<system-err><![CDATA[]]></system-err>
	</testcase>
</testsuite>
